sold.cc

```diff
@@ -1223,11 +1225,16 @@ class Sold {
             }
         }
 
-        // TODO(hamaji): From ld.so.conf. Make this customizable.
+        // TODO(hamaji): Make this customizable.
+        library_paths.push_back("/lib");
```

Comment:
I think now we can remove most of these hard coded values?

As per `ld-linux`, we should copy ldsoconfs first, followed by `/lib` and `/usr/lib`?